.github/workflows/cleanup-versions-workflow.yml

```diff
@@ -8,15 +8,16 @@ jobs:
   cleanup:
     runs-on: ubuntu-latest
     steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
       - name: Setup gcloud
-        uses: GoogleCloudPlatform/github-actions/setup-gcloud@0.1.1
+        uses: google-github-actions/setup-gcloud@master
```

Comment:
@samthor @MichaelSolati should we switch this to 0.2.0? https://github.com/google-github-actions/setup-gcloud/releases

I think OSPO policy is that if we use actions we have to use a pinned version and not master

Reply:
Even if it's not I guess it'd also be good for potential breaking changes, I didn't consider that...